core/src/test/scala/unit/kafka/admin/ReassignPartitionsServiceClientTest.scala

```diff
@@ -0,0 +1,394 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package unit.kafka.admin
+
+import java.util.concurrent.ExecutionException
+import java.util.{Collections, Properties}
+
+import kafka.admin.{AdminClientReassignCommandService, AdminOperationException, BrokerMetadata, RackAwareMode, ZkClientReassignCommandService}
+import kafka.log.LogConfig.{FollowerReplicationThrottledReplicasProp, LeaderReplicationThrottledReplicasProp}
+import kafka.server.{ConfigType, KafkaServer}
+import kafka.utils.{Logging, TestUtils}
+import kafka.zk.{AdminZkClient, KafkaZkClient, ZooKeeperTestHarness}
+import org.apache.kafka.clients.admin.{Admin, MockAdminClient}
+import org.apache.kafka.common.errors.UnknownTopicOrPartitionException
+import org.apache.kafka.common.{Node, TopicPartition, TopicPartitionInfo}
+import org.easymock.EasyMock.{replay, _}
```

Comment:
There is an initiative to use `Mockito` as Kafka's mocking library, so we shouldn't introduce new tests with EasyMock